bluetooth: services: nus: Add nus-client context upon msg reception. (#6034)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-changelog.rst

This section provides detailed lists of changes by :ref:`library <libraries>`.

Bluetooth libraries and services
--------------------------------

Updated:

* :ref:`nus_client_readme` library:

>   * Added context to :c:func:`bt_nus_client.received` to enable using it in a multi-client application.
>   * Added context to :c:func:`bt_nus_client.sent` to enable using it in a multi-client application.
>   * Added context to :c:func:`bt_nus_client.unsubscribed` to enable using it in a multi-client application.

Reviewer: ```suggestion
  * Added context to functions :c:func:`bt_nus_client.received`, :c:func:`bt_nus_client.sent` and :c:func:`bt_nus_client.unsubscribed` to enable their use in a multi-client application.
```

Author: Sounds reasonable. Updated!
